projects/lib/src/chessgame.cpp

```diff
@@ -296,7 +301,15 @@ void ChessGame::onMoveMade(const Chess::Move& move)
 void ChessGame::startTurn()
 {
 	if (m_paused)
+	{
+		if (!playerToMove()->isHuman())
```

Comment:
You don't have to check if the player is human; just call `playerToMove()->stopPondering()` like you do in the `halt()` method.

Reply:
Ah right, `ChessPlayer::stopPondering()` saves this. So no need to test for `isHuman()`.